tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -18,16 +18,17 @@ spec:
   params:
     - description: A docker image of operator-pipeline-images for the steps to run in.
       name: pipeline_image
-      default: "quay.io/redhat-isv/operator-pipelines-images:released"
+      default: "quay.io/jluza/release-service-utils:latest"
       type: string
-    - description: Manifest digest for the signed content, usually in the format sha256:xxx
-      name: manifest_digest
+    - description: |
+        List of space separated manifest digests for the signed content, usually in the format sha256:xxx
+        e.g. sha256:xxx sha256:yyy
+      name: manifest_digests
       type: string
-    - description: Docker reference for the signed content, e.g. registry.redhat.io/redhat/community-operator-index:v4.9
-      name: reference
-      type: string
-    - description: Name of the user that requested the signing, for auditing purposes
```

Comment:
Why are we removing the requester?

Reply:
Requester is taken from client certificate (CN or UID)

Reply:
ProdSec wants to know which RH user has requested the Release and thus we have to propagate that to the signing process. Knowing the CN or UID is not sufficient for ProdSec according to what we agreed to.

Reply:
Ok, that makes sense.